docs/plugins/outputs/riak.asciidoc

```diff
@@ -180,12 +165,10 @@ here: https://github.com/basho/riak-ruby-client/wiki/Connecting-to-Riak
 You'll likely want something like this:
 
 [source, ruby]
------
     ssl_opts => {
        "pem" => "/etc/riak.pem"
        "ca_path" => "/usr/share/certificates"
     }
------
 
```

Comment:
REGRESSION:  Additional formatting required for proper rendering with --asciidoctor